README.md

```diff
@@ -47,7 +47,8 @@ When the control panel is enabled:
 1. Select the **GeometryManager** GameObject in the Hierarchy window.
 2. Click on the **Load New Model** button in the Inspector window.
 3. Navigate to **"Assets/Models/"** and select one of the \*.gltf files.
-4. **Enjoy** your freshly loaded model!
+4. Go to **"Assets/Scenes/"** and select and load the scene SteamAudio.
+5. Press play and **enjoy** your freshly loaded model!
```

Comment:
There is no explanation in the file about changing a material.

Reply:
Noted!